apps/api/src/server/applications/application/application.validators.js

```diff
@@ -86,13 +86,13 @@ const timestampToDate = (value) => {
 
 export const validateCreateUpdateApplication = composeMiddleware(
 	body('geographicalInformation.gridReference.easting')
-		.isLength({ min: 6, max: 6 })
```

Comment:
@backendTeam: apparently  toInt() should be before the isLength() because otherwise something like
'abcde1' would pass the test